api/capi/src/nnstreamer-capi-util.c

```diff
@@ -97,22 +97,45 @@ ml_tensors_info_initialize (ml_tensors_info_s * info)
   return ML_ERROR_NONE;
 }
 
+/**
+ * @brief Gets the tensor rank.
+ */
+static guint
+ml_tensor_info_get_rank (const ml_tensor_info_s * info)
+{
+  guint i;
+
+  if (!info)
```

Comment:
Did we change the error checking code following this style to ```info == NULL```, didn't we?

Reply:
updated.
TODO need to fix coding style later